cmd/storage/internal/dal/dal.go

```diff
@@ -75,6 +75,91 @@ func (r *repo) schemaLock(f func() error) func() error {
 	}
 }
 
+func (r *repo) User(login string) (user app.UserData, err error) {
+	err = r.tx(ctx, nil, func(tx *sqlxx.Tx) error {
+		var res []resUser
+		err := tx.NamedSelectContext(ctx, &res, sqlGetUser, argGetUser{
+			Login: login,
+		})
+		if err != nil {
```

Comment:
switch {
		case errors.Is(err, sql.ErrNoRows):
			return app.ErrNotFound
		case err != nil:
			return err
		}